### PR TITLE
Added AnuragThePathak.json

### PR DESCRIPTION
**Pull request by AnuragThePathak**
Learnt how to contribute to other's repository form the Code for Cause video and hence practicing.

### CODE REVIEW

**Review by bishalr0y — COMMENTED** [review]
LGTM🚀

**Comment by AnuragThePathak** [comment]
@bishalr0y Sorry on the right top below your profile picture github was showing something as "Still in progress? Convert to draft" . I thought that was your and hence converted to draft clicking that.

File: src/data/contributors/AnuragThePathak.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (shreyshreyansh) [inline src/data/contributors/AnuragThePathak.json:1]: LGTM

Comment (shreyshreyansh) [reply src/data/contributors/AnuragThePathak.json:1]: Should merge it

**Pull request closed without merging**